kolibri/plugins/learn/assets/src/views/LearningActivityBar.vue

```diff
@@ -120,13 +119,14 @@
         required: true,
       },
       /**
-       * Learning activity constant
+       * An array of one or more learning activities constants
        */
-      learningActivityKind: {
-        type: String,
+      learningActivities: {
+        type: Array,
```

Comment:
I didn't consider important to support both `Array` and `String` like for  `LearningActivityIcon` which is much more general component than `LearningActivityBar` so I've just changed to `Array` only here. Another reason is that I assume that we will simply pass data as received from API to the bar which is always an array, even when there is only one item.